tests/manager/api/admin/controller/test_collections.py

```diff
@@ -108,13 +109,24 @@ def test_collections_get_with_no_collections(
         expected_names = {k for k, v in controller.registry}
         assert names == expected_names
 
+    @pytest.mark.parametrize(
+        "is_inactive",
+        (
+            pytest.param(True, id="inactive collection"),
+            pytest.param(False, id="active collection"),
+        ),
+    )
     def test_collections_get_collections_with_multiple_collections(
         self,
         controller: CollectionSettingsController,
         flask_app_fixture: FlaskAppFixture,
         db: DatabaseTransactionFixture,
+        is_inactive: bool,
     ) -> None:
-        [c1] = db.default_library().associated_collections
+        default_library = db.library(short_name="default", name="Default Library")
```

Comment:
Minor: This could be kind of confusing. I tend to assume the "default library" is what comes back from `db.default_library()`. Might be helpful to either:
- Just use `db.default_library()`
- use `db.library()` but rename the variable so its not called `default_library`

Reply:
I don't think I can use `db.default_library()` here because of its side effects. I'll change the variable name.